notebooks/validation/osm_vs_entso_comparison_for_base_network.ipynb

```diff
@@ -0,0 +1,295 @@
+      "text/plain": [
+       "<Figure size 1500x500 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.figure(figsize=(15, 5))\n",
+    "osm_x_axis = np.arange(len(osm_voltage.v_nom))\n",
+    "entsoe_x_axis = np.arange(len(entsoe_voltage.v_nom))\n",
+    "\n",
+    "plt.bar(osm_x_axis - 0.4, osm_voltage.length, 0.4, label='osm')\n",
+    "plt.bar(entsoe_x_axis, entsoe_voltage.length, 0.4, label='entsoe')\n",
+    "plt.xticks(osm_x_axis, osm_voltage.v_nom)\n",
+    "plt.xticks(rotation=0)\n",
+    "plt.title('Intercomparison of length_in_km between OSM and ENTSOE data based on voltage values')\n",
+    "plt.xlabel('voltage_values(kV)')\n",
+    "plt.ylabel('Length(km)')\n",
+    "plt.grid('True')\n",
+    "plt.legend()\n",
+    "plt.show()\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Conclusion\n",
+    "\n",
+    "From the comparison above, it is observed that there is about 3x discrepancies between osm data and entsoe data. To understand the cause of the discrepancy, another notebook is being used for investigating the network using base network files for AT and MK.\n",
```

Comment:
Agree that it absolutely make sense to refer AT-MK notebook. However, explanation seems to be missed for the reason for OSM-ENTSOE discrepancy for `base.nc`-`elec.nc`. You may go trough our discussion in Discord channel to remember the details.